### PR TITLE
 Added guide for GPU on WSL2

### PR DESCRIPTION
**Pull request by Avditvs**
Added guide for GPU usage in Windows Subsystem For Linux

### CODE REVIEW

File: wsl2_gpu_guide/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# WSL2 gpu guide
```
Comment (angerson) [inline wsl2_gpu_guide/README.md:1]: Can you change this to match up with the README? That is, the README should also say "WSL2 GPU Guide" (the directory name) and the short description should be the same as well. The [example](https://github.com/tensorflow/build/tree/master/directory_template) includes all that too -- the long description will fit under the section separator (`***`)

File: wsl2_gpu_guide/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+The directory contains instructions for enabling GPU with Tensorflow on a WSL2 virtual machine.
+
+Maintainer: @Avditvs (Independant)
```
Comment (angerson) [inline wsl2_gpu_guide/README.md:5]: Minor typo: "independent"

**Review by angerson — CHANGES_REQUESTED** [review]
Very nice! Thank you for adding this!

Once the formatting has been adjusted I'll approve and merge!

**Comment by Avditvs** [comment]
@angerson The changes have been applied.

**Review by angerson — APPROVED** [review]
Thanks!

**Pull request merged**